spec/lib/bookmark_query_spec.rb

```diff
@@ -119,19 +119,19 @@ def bookmark_query(user: nil, params: nil)
     context "for a whispered post" do
       before do
         post_bookmark.bookmarkable.update(post_type: Post.types[:whisper])
-        SiteSetting.enable_whispers = true
+        SiteSetting.whispers_allowed_groups = "#{Group::AUTO_GROUPS[:staff]}"
```

Comment:
Worth moving this into a helper like `allow_staff_to_whisper` or similar?

Reply:
Do we need all of those now that the `staff` bypass has been restored? 🤔

Reply:
The staff bypass only takes effect if whispers are enabled, which means at least a group is set.